pgvectorscale/src/access_method/start_nodes.rs

```diff
@@ -0,0 +1,125 @@
+use pgvectorscale_derive::{Readable, Writeable};
+use rkyv::{Archive, Deserialize, Serialize};
+
+use crate::access_method::labels::{Label, LabelSet};
+use crate::access_method::node::{ReadableNode, WriteableNode};
+use crate::util::{ItemPointer, ReadableBuffer, WritableBuffer};
+use pgrx::PgRelation;
+use std::collections::BTreeMap;
+
+use super::labels::LabelSetView;
+
+const OVERLOAD_THRESHHOLD: usize = 20;
+
+/// Start nodes for the graph.  For unlabeled vectorsets, this is a single node.  For
+/// labeled vectorsets, this is a map of labels to nodes.
+#[derive(Clone, Debug, PartialEq, Eq, Archive, Deserialize, Serialize, Readable, Writeable)]
+#[archive(check_bytes)]
+pub struct StartNodes {
+    /// Default starting node for the graph.
+    default_node: ItemPointer,
+    /// Labeled starting nodes for the graph
+    labeled_nodes: BTreeMap<Label, ItemPointer>,
+    /// Load for a start node
+    node_count: BTreeMap<ItemPointer, usize>,
```

Comment:
This is the number of labels a start node corresponds to? We should either improve name or make a better comment (or both)